editor/dependency_editor.cpp

```diff
@@ -146,7 +150,17 @@ void DependencyEditor::_fix_all() {
 	for (KeyValue<String, HashMap<String, String>> &E : candidates) {
 		for (const KeyValue<String, String> &F : E.value) {
 			if (!F.value.is_empty()) {
+				String sub_res = "";
```

Comment:
```suggestion
				String sub_res;
```

Reply:
And all others, there's more further in the diff, the best is to search for ` = "";` in your diff and remove the unnecessary ones.